pkg/nsgenerator/random_ns_generator.go

```diff
@@ -0,0 +1,22 @@
+package nsgenerator
```

Comment:
[blocking]

The prefix `ns` may not be known by new contributors so proposal to rename this back to `utils` or `namespace` with a slight preference for `namespace`.


If we use `namespace`, we  may need to use an [import alias](https://scene-si.org/2018/01/25/go-tips-and-tricks-almost-everything-about-imports/) if the file already uses nmt's namespace.

Reply:
`namespace` it's!